src/components/Comments.vue

```diff
@@ -4,57 +4,60 @@
       <i class="el-icon-s-comment"></i>
       {{ $t('comments.title') }}
     </div>
-    <div
-      v-for="(comment, index) in comments"
-      :key="index"
-      class="py-3 grid grid-cols-3 sm:grid-cols-4 md:grid-cols-8 border-b"
-      :class="index === comments.length - 1 ? 'border-none pb-0' : ''"
-    >
-      <div id="user-area" class="h-full col-span-1">
-        <img
-          class="w-full rounded-full"
-          :src="getAvatar(get(comment, 'relationships.creator[0].avatar'))"
-        />
-      </div>
+    <transition-group name="fade" tag="div">
       <div
-        id="comment-area"
-        class="pl-4 col-span-full col-start-2 place-content-start"
+        v-for="(comment, index) in comments"
+        :key="comment.id"
+        class="py-3 grid grid-cols-3 sm:grid-cols-4 md:grid-cols-8 border-b"
+        :class="index === comments.length - 1 ? 'border-none pb-0' : ''"
       >
-        <div class="font-medium text-indigo-500 flex justify-between">
-          <span>
-            {{ getFullName(get(comment, 'relationships.creator[0]')) }}
-            <span class="text-gray-500">
-              {{ $t('comments.userCommented.text') }}
-            </span>
-          </span>
-          <el-dropdown
-            v-if="
-              get(comment, 'relationships.creator[0].id', null) === userState.id
-            "
-          >
+        <div id="user-area" class="h-full col-span-1">
+          <img
+            class="w-full rounded-full"
+            :src="getAvatar(get(comment, 'relationships.creator[0].avatar'))"
+          />
+        </div>
+        <div
```

Comment:
Please consider extracting a `Comment` component which will display a single comment including actions and avatar, comment etc

Reply:
e un caz particular de afisare a comentariilor unui blog, cu informatii despre autor, din cauza asta nu cred că am putea folosi o componenta reutilizabila pentru afisarea fiecărui comentariu, pentru ca în alte cazuri informația primită de la server ar arata altfel, în cazul de față pentru a primi comentariile unui blog, requestul arata așa:
![image](https://user-images.githubusercontent.com/28743656/132236002-05f3e42c-4fd0-41d9-bac6-a26958c80e91.png)